test/src/databases/duckdb/materialization.spec.ts

```diff
@@ -33,12 +33,12 @@ describe.each(allDucks.runtimeList)('duckdb:%s', (dbName, runtime) => {
     `;
 
     const qm = runtime.loadQuery(query, {replaceMaterializedReferences: true});
-    const pq = await qm.getPreparedQuery();
+    const preparedResult = await qm.getPreparedResult();
 
-    expect(pq.preparedResult.sql).toBe(
+    expect(preparedResult.sql).toBe(
```

Comment:
Note here we get the prepared result still in runtime world.

Doing `qm.getPreparedQuery().preparedResult` would end up not using the setting.

Reply:
should we deprecate the getter at some point?

Reply:
Yeah probably. We could even just add a `@deprecated` to the doc string right now. My event logging PR also adds a setting to `get preparedResult`/`getPreparedResult()`.